Declaring Java interface deprecation (#2365)
oneapi-src/oneDAL: INSTALL.md

            call "C:\Program Files (x86)\Intel\oneAPI\compiler\latest\env\vars.bat"

> 4. Deprecation Notice: Java interfaces in the oneDAL library have been deprecated and may no longer be supported in future releases.

Reviewer: ```suggestion
4. Deprecation Notice: The Java interfaces are deprecated in the oneDAL library and may no longer be supported in future releases.
```
